Common/otoolstuff.cpp

```diff
@@ -1,29 +1,26 @@
 #include "otoolstuff.h"
 #include "stringutils.h"
-#include "rbridge.h"
 #include <fstream>
 #include <boost/algorithm/string/predicate.hpp>
 #include "utils.h"
 #include "appinfo.h"
 #include <iostream>
+#include <filesystem>
 
 std::string _system(std::string cmd)
 {
-	const char *root, *relativePath;
-
-	if (!rbridge_requestTempFileName("log", &root, &relativePath))
```

Comment:
Not that im saying you cant just use temp_directory_path, but im curious as to why.